src/pages/about-details/index.js

```diff
@@ -0,0 +1,77 @@
+import { StyledAbout } from "./styled"
+import NavBar from '../../components/navbar'
+
+const PageAbout  = () => {
+  return(
+    <>
+    <NavBar/>
+    <StyledAbout>
+    <h2 className="about-title">
+      Sobre Nossa Casa
+      </h2>
+       <div className="content-container">
+                <div className="topic">
+                  <h3>Nossa história até aqui</h3>
+                  <p>
+                    Em 2018, a Nossa Casa nasce com o intuito de oferecer à
+                    população acesso a serviços, produtos e atividades nas áreas, produtos e atividades nas áreas
```

Comment:
Acho que uma parte do texto se repetiu

```suggestion
                    população acesso a serviços, produtos e atividades nas áreas
```